pkg/plugin/plugin.go

```diff
@@ -238,7 +254,16 @@ func CmdAdd(args *skel.CmdArgs) error {
 		return err
 	}
 
-	brIface, err := setupBridge(netconf.BrName)
+	bridgeName, err := getBridgeName(netconf.BrName,ovnPort)
+	if err != nil {
+		return err
+	}
+
+	if bridgeName == "" {
```

Comment:
will this ever happen?

if `bridgeName == "" && ovnPort != ""`, then we return error in previous section. Tell me if I am missing something.

Reply:
Good one